proto/src/determined/api/v1/command.proto

```diff
@@ -32,8 +32,11 @@ message GetCommandsRequest {
   int32 offset = 3;
   // Limit the number of commands. A value of 0 denotes no limit.
   int32 limit = 4;
-  // Limit commands to those that are owned by the specified users.
-  repeated string users = 5;
+  // Limit commands to those that are owned by users with the specified userIds.
+  repeated int32 user_ids = 6;
+  // Limit commands to those that are owned by users with the specified
+  // usernames.
+  repeated string users = 7;
```

Comment:
I don't think you need to make a new users field here, you can change the comment without changing the type

Reply:
the intent here was to make users use `7` so that its argument would come later in the function defined in `api-ts-sdk/api.ts`, and then `detApi.Commands.getCommands` could be called with the last `users` argument omitted.

Reply:
I don't think that this is a good way of handling users argument being optional or less-used (?) -- if the API added a new field in the future then it would negate the benefit of changing users like this

Reply:
that's a good point. i wanted to avoid using `undefined` as the value for `users` but i guess if we're keeping both fields we might as well be explicit about the fact that we're ignoring one of them.